MobileWallet/TariLib/Tor/OnionManager.swift

```diff
@@ -233,7 +233,7 @@ public class OnionManager: NSObject {
                         var completeObserver: Any?
                                                 
                         completeObserver = self.torController?.addObserver(forCircuitEstablished: { established in
-                            print("self.torController?.isConnected: \(self.torController?.isConnected)")
+                            print("self.torController?.isConnected: \(String(describing: self.torController?.isConnected))")
```

Comment:
Was a debug message I meant to actually remove before committing

Reply:
Have removed it.